api/response/schema.py

```diff
@@ -16,26 +17,40 @@ class Meta:
 
 class CreateResponse(graphene.Mutation):
     class Arguments:
-        question_id = graphene.Int(required=True)
+        question_id = graphene.List(graphene.Int, required=True)
         room_id = graphene.Int(required=True)
         rate = graphene.Int()
         text_area = graphene.String()
         missing_items = graphene.List(graphene.Int)
 
-    response = graphene.Field(Response)
+    response = graphene.List(Response)
 
     def mutate(self, info, **kwargs):
         validate_empty_fields(**kwargs)
         query = Room.get_query(info)
         room = query.filter_by(id=kwargs['room_id']).first()
         if not room:
             raise GraphQLError("Non-existent room id")
-        get_question = Question.query.filter_by(id=kwargs['question_id']).first() # noqa
-        if not get_question:
-            raise GraphQLError("Question does not exist")
-        question_type = get_question.question_type
-        resource = create_response(question_type, **kwargs)
-        return CreateResponse(response=resource)
+        question_ids = kwargs.pop('question_id')
+        resources = []
```

Comment:
From what is going on in the code, I would think this should be `response` and not `resource`. Same for line 42. Kindly share your thoughts on this, @kampkelly.

Reply:
Thanks @RonKbS for the feedback. 
This has been renamed to response and other places accordingly.